appveyor.yml

```diff
@@ -4,6 +4,20 @@
 
 ## General configuration
 version: '{build}'
+
+only_commits:
+  files:
+    - bin/
+    - lib/
+    - test/
+    - .mocharc.yml
+    - .wallaby.js
+    - .appveyor.yml
+    - index.js
+    - package.json
+    - package-lock.json
```

Comment:
More a question than anything: [`package-lock.json` can't be published](https://docs.npmjs.com/files/package-lock.json), so why put it here? For compiled output?

Reply:
We are using `npm ci` on our CI servers to install the exact dependencies noted in `package-lock.json`. This means a build must be triggered if the file changes because changes in dependencies can affect test outcomes. This has nothing to do with what runs on a consumers installation